tests/utils/test_store.py

```diff
@@ -156,7 +158,7 @@ def test_check_for_blocked_document_raises_error(store, mock_boto, blocked_value
         store.check_for_blocked_document("service-id", "doc-id")
 
 
-def test_check_for_blocked_document_raises_expired_error(store, expired_document):
+def test_check_for_blocked_document_delete_marker_document_expired_error(store, delete_markered_document):
```

Comment:
This name is pretty hard to parse (although it's obvious enough what it does from the content so maybe that's fine)

Reply:
I know, but I was naming things systematically and this is just an extreme case.